### DISCUSSION TITLE
明细表怎么选中整个表格呢？比如说在左上角点击选中整个表格，可复制

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [antvis/S2#1775] [Q&A] [answered]
明细表怎么选中这个表格呢？比如说在左上角点击选中整个表格，可复制，求大神指导下，有提供什么api 方法之类的不？

**Comment by lijinke666** [comment, marked as answer]
selectAll: https://s2.antv.vision/zh/docs/api/basic-class/interaction

可参考 https://s2.antv.vision/zh/examples/case/data-preview#index

**Reply by Chasen-Zhang** [reply]
谢谢

**Reply by Chasen-Zhang** [reply]
选中之后，鼠标一移动就失去了选中效果，体验上感觉不太好

**Reply by Chasen-Zhang** [reply]
@lijinke666 请教下，有什么好的方式吗
